web/themes/custom/atrium/templates/node/node--srl-howto-instruction--full.html.twig

```diff
@@ -35,13 +35,80 @@
     </div>
   {% endif %}
 
-  {% set right_column = '' %}
-  {% if content.field_blocks|render or content.field_forms|render %}
-    {% set right_column %}
+  {% set right_column %}
+    <div class="mobile:display-none jcc-share--desktop">
+      {% include '@molecules/blocks/share/share.twig' with {
+        social_links: {
+          items: [
+            {
+              id: "print",
+              title: "Print",
```

Comment:
I don't see a `title` field in the template.

Reply:
May have been in the original UI component. Will revisit to determine if we need to restore title.

Reply:
FYI... title does not seem to have been used at all in the template since first created:
https://github.com/JudicialCouncilOfCalifornia/courtyard/blob/a4530b059291609091854964171dd0b1b01daec9/source/_patterns/02-molecules/blocks/share/share.twig

Will remove from SRL-282 and file a to-do for courtyard to address.